lib/msf/core/post/file.rb

```diff
@@ -362,10 +362,23 @@ def read_file(file_name)
     session.shell_command_token("while read line; do echo $line; done <#{file_name}")
   end
 
+  # Checks to see if there are non-ansi or newline characters in a given string
+  #
+  # @param data [String] String to check for non-ansi or newline chars
+  # @return bool
+  def is_safe?(data)
```

Comment:
This will be included in all modules, do we want to rename it to something else - or move the code elsewhere so it's not mixed in unintentionally

Reply:
Moved and renamed